docs/reference/data-frames/apis/put-transform.asciidoc

```diff
@@ -56,9 +56,11 @@ IMPORTANT:  You must use {kib} or this API to create a {dataframe-transform}.
   and optionally a `pipeline` id. See <<data-frame-transform-dest>>.
 
 `frequency`::
-  (time units) Optional. The interval between checks for changes in the source indices
-  when the {dataframe-transform} is running continuously. Defaults to `1m`.
-  The lowest permitted value is `1s`; the highest `1h`.
+  (time units) Optional. The interval between checks for changes in the source
+  indices when the {dataframe-transform} is running continuously. If there is
```

Comment:
From discussions with @sophiec20 about the difference between data frame transforms with one or many checkpoints, I'm wondering whether this qualification is necessary.

```suggestion
  indices. If there is
```

Reply:
1. `when .... is running continuously` -- yes, this is valid here.
2. `Also determines the retry interval in the event of transient failures during {dataframe-transform} search or indexing.` -- we recently added a retry to indexing, best to mention both